src/Rules/Missing/MissingReturnRule.php

```diff
@@ -64,6 +64,10 @@ public function processNode(Node $node, Scope $scope): array
 		} elseif ($scopeFunction !== null) {
 			$returnType = ParametersAcceptorSelector::selectSingle($scopeFunction->getVariants())->getReturnType();
 			if ($scopeFunction instanceof MethodReflection) {
+				if ($scopeFunction->isAbstract()) {
```

Comment:
This is not necessary, but it's more explicit. Current detection is hidden in `NodeScopeResolver` and it's based on internal AST structure.